README.md

```diff
@@ -27,13 +35,19 @@ npm start
 serving "app" at http://0.0.0.0:8080
 ```  
 
-Go to the URL listed in the output (`http://0.0.0.0:8080` in the example above) to see your page. Hit "refresh" when you make any changes to the files on disk.
+# If npm start returns an error it is likely due to an older version of npm, try running:
```

Comment:
This text is a bit misleading, since the command you suggest will not update the version of `npm` itself. How about:

### If `npm start` returns an error
You may need to update the dependencies.
```
rm -rf node_modules && npm install
```

Reply:
I'm confused by this. I checked my history and all I ran to fix my issue was npm update. I checked and my npm was on version 4.2.0. I then ran the command you suggested and after it when I run npm start I get an error again (likely the same as the last error I got, but I didn't copy that one so I can't be sure). Here's the error:

```
Matts-Mac-mini:cornellfencing.github.io miniparty$ npm start

> cornellfencingv4@1.0.0 start /Users/miniparty/Documents/CornellFencing/temp/cornellfencing.github.io
> static -a 0.0.0.0 -H '{"Cache-Control": "no-cache, must-revalidate"}' app

sh: static: command not found

npm ERR! Darwin 16.7.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v7.8.0
npm ERR! npm  v4.2.0
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! cornellfencingv4@1.0.0 start: `static -a 0.0.0.0 -H '{"Cache-Control": "no-cache, must-revalidate"}' app`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the cornellfencingv4@1.0.0 start script 'static -a 0.0.0.0 -H '{"Cache-Control": "no-cache, must-revalidate"}' app'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the cornellfencingv4 package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     static -a 0.0.0.0 -H '{"Cache-Control": "no-cache, must-revalidate"}' app
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs cornellfencingv4
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls cornellfencingv4
npm ERR! There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/miniparty/.npm/_logs/2018-03-09T03_47_07_856Z-debug.log
```

After this I run update and get the following output
```
Matts-Mac-mini:cornellfencing.github.io miniparty$ npm update
cornellfencingv4@1.0.0 /Users/miniparty/Documents/CornellFencing/temp/cornellfencing.github.io
├── gh-pages@0.12.0 
└── node-static@0.7.10 
```

Then it all works again. I'm thinking maybe the error is in me saying "likely due to an older version of npm" rather than "likely due to a dependency of npm needing updating". Do you agree?

Reply:
> Then it all works again. I'm thinking maybe the error is in me saying "likely due to an older version of npm" rather than "likely due to a dependency of npm needing updating". Do you agree?

Basically, yes. 😄  `npm` is a package manager, so our problem here is not an older version of `npm` or an older version of a dependency of `npm`. It's just the project packages being out of date. So I think the text I suggested in my original comment above makes sense.